### PR TITLE
Readme: Mention message definition update script

### PR DESCRIPTION
**Pull request by mortenfyhn**
Hi all, I was reading the PX4 dev docs on RTPS/ROS2, and I felt that mentioning more specifically how the ROS message definitions are generated in this repo's readme could be useful. Especially (maybe exclusively) useful for those who run custom PX4 forks with modified uORB. I also added a direct link to the script on the PX4/Firmware repo at master. This link will of course break if the script moves, but hopefully will almost never happen.

Let me know if this is an addition you want or not, or if you want any changes to the wording.

I'm not sure about your process for updating the ros1 branch when non-message definitions change on the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,7 +10,9 @@ The uORB message definitions, which represent the counter-part of the ROS2 messa
 
 ## How are these messsage definitions generated?
 
-The PX4 Firmware Jenkins-based CI/CD deploys new messages or modifications to the current definitions with each build of the master branch of the PX4 Firmware. A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tide coupling between the message definitions.
+When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by a script [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py). You can also use this script to generate your own ROS message definitions, for instance if you have a custom PX4 firmware fork with modified uORB definitions.
```
Comment (TSC21) [inline README.md:13]:
```suggestion
When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

File: README.md [hunk 2]
```diff
@@ -10,7 +10,9 @@ The uORB message definitions, which represent the counter-part of the ROS2 messa
...
+
+A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tidy coupling between the message definitions.
```
Comment (TSC21) [inline README.md:15]: We should remove this from the README and put it under an issue instead. Can you please remove it? Thanks!

File: README.md (hunk 1 above)
Comment (mortenfyhn) [reply README.md:13]: I agree. Are you okay with keeping the "you can" instead of "one can"? I feel it's more clear/direct.

File: README.md (hunk 2 above)
Comment (mortenfyhn) [reply README.md:15]: Absolutely!

File: README.md (hunk 1 above)
Comment (TSC21) [reply README.md:13]: On this kind of things I do prefer the 3rd person.

Comment (mortenfyhn) [reply README.md]: :+1:

**Comment by mortenfyhn** [comment]
Probably a good idea to squash this when done.

File: README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ The uORB message definitions, which represent the counter-part of the ROS2 messa
 
 ## How are these messsage definitions generated?
 
-The PX4 Firmware Jenkins-based CI/CD deploys new messages or modifications to the current definitions with each build of the master branch of the PX4 Firmware. A future improvement will also include tagging and releasing this repository according to the tags/releases of the PX4 Firmware, for a more tide coupling between the message definitions.
+When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository changes, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```
Comment (TSC21) [inline README.md:13]:
```suggestion
When uORB message definitions in the [PX4 firmware](https://github.com/PX4/Firmware) repository change, a Jenkins-based CI/CD automatically generates and pushes updated ROS message definitions to this repository. The ROS message definitions are generated by the [`uorb_to_ros_msgs.py`](https://github.com/PX4/Firmware/blob/master/msg/tools/uorb_to_ros_msgs.py) script. One can also use this script to generate its own ROS message definitions for new or modified uORB messages.
```

Comment (TSC21) [reply README.md:13]: It's not in Jenkins anymore, but rather in Github Actions.

Comment (mortenfyhn) [reply README.md]: Aha! I found it still in the Jenkinsfile, but maybe that's just kept around but not running. And thanks for spotting the typo.

**Comment by TSC21** [comment]
@mortenfyhn can you squash it all in the same commit?

**Comment by mortenfyhn** [comment]
> @mortenfyhn can you squash it all in the same commit?

Done.

**Comment by mortenfyhn** [comment]
Great! Thanks for the very quick review.

**Pull request merged**